packages/spanner/src/transaction-request.js

```diff
@@ -201,6 +202,12 @@ TransactionRequest.prototype.createReadStream = function(table, query) {
     table: table
   }, query);
 
+  if (this.transaction && this.id) {
```

Comment:
Haha, all the code within `TransactionRequest` was originally baked into our `Table` class - we didn't tackle transactions until later so a lot of that code was refactored out into what is now `TransactionRequest`, which serves as an abstract class for `Table` and `Transaction`. I think this simply just got missed while refactoring the code originally.